source/extensions/tracers/opencensus/opencensus_tracer_impl.cc

```diff
@@ -290,11 +299,15 @@ Driver::Driver(const envoy::config::trace::v3::OpenCensusConfig& oc_config,
       opts.address = oc_config.ocagent_address();
     } else if (oc_config.has_ocagent_grpc_service() &&
                oc_config.ocagent_grpc_service().has_google_grpc()) {
+#ifdef ENVOY_GOOGLE_GRPC
       const envoy::config::core::v3::GrpcService& ocagent_service =
           oc_config.ocagent_grpc_service();
       auto channel = Envoy::Grpc::GoogleGrpcUtils::createChannel(ocagent_service, api);
       opts.trace_service_stub =
           ::opencensus::proto::agent::trace::v1::TraceService::NewStub(channel);
+#else
+      throw EnvoyException("Opencensus tracer: cannot handle stackdriver grpc option, google grpc is not built in.");
```

Comment:
Thanks. Looks like it needs a format fix (from CI). Also, maybe consider adding a test for this throw. It would only show up on `--define google_grpc=disabled` builds, but we have one of these in CI `bazel.compile_time_options`.

Reply:
Done.